proto/server/pi_server.cpp

```diff
@@ -69,6 +70,17 @@ StreamChannelClientMgr *packet_in_mgr;
 void packet_in_cb(DeviceMgr::device_id_t device_id, std::string packet,
                   void *cookie);
 
+
+// DeviceMgr::Status == google::rpc::Status
+grpc::Status make_status(const DeviceMgr::Status &from) {
```

Comment:
nit: Consider "to_grpc_status" or "grpc_status" to make it clear what 'status' type it is converting to.